packages/edit-site/src/components/site-hub/index.js

```diff
@@ -156,25 +156,32 @@ const SiteHub = memo( ( { isTransparent, className } ) => {
 							<div className="edit-site-site-hub__title">
 								{ decodeEntities( siteTitle ) }
 							</div>
+							<HStack
+								spacing={ 0 }
+								expanded={ false }
+								style={ { flexShrink: '0' } }
```

Comment:
Worth adding a class for this, or okay inline?

Reply:
I don't know, I'd say just add a class but I don't care much either.

Reply:
I suppose a class might be handy in the future...